### PR TITLE
New cheatsheet: qbittorrent

### PR DESCRIPTION
**Pull request by devanshdalal**
---

Instant Answer Page: https://duck.co/ia/view/qbittorrent_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@devanshdalal Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/qbittorrent.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
+  "id": "qbittorrent_cheat_sheet",
+  "name": "qbittorrent",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/qbittorrent.json]: Lets have the name "qBittorrent"

File: share/goodie/cheat_sheets/json/qbittorrent.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  "description": "Keyboard shortcuts for qBittorrent",
+  "metadata": {
+    "sourceName": "qBittorrent Keyboard Shortcuts | KB Mode",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/qbittorrent.json]: "KB Mode" would be enough here

File: share/goodie/cheat_sheets/json/qbittorrent.json (hunk 1 above)
Comment (devanshdalal) [inline share/goodie/cheat_sheets/json/qbittorrent.json]: I am not exactly sure about this, If I remember correctly, my initial build tests failed saying you cannot have CAPs in name field.

**Comment by devanshdalal** [comment]
Hi

I want t know the status on this. Can anyone help me out on this one?

Thanks

**Comment by jophab** [comment]
@tagawa Please review this.

**Comment by pjhampton** [comment]
Closing due to inactivity. 

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**